Configuration/Geometry/README.md

```diff
@@ -35,6 +36,7 @@ Several detector combinations have been generated:
 * D25 = T6+C3+M2+I4+O2+F2 
 * D26 = T9+C3+M2+I1+C2+F2
 * D27 = T10+C3+M2+I1+O2+F2
+* D28 = T10+C4+M2+I1+O2+F3
```

Comment:
D27 is a test scenario for tracker, so D28 should not be based on that one. I would start from D21 (which is D17 with the T6 tracker update) and just swap C3 with C4. (Also, there is no F3 - a typo?)